File-Formats/DocIO/Create-Word-document-in-ASP-NET-Core.md

```diff
@@ -14,15 +14,15 @@ Syncfusion Essential DocIO is a [.NET Core Word library](https://www.syncfusion.
 
 Step 1: Create a new ASP.NET Core Web application project.
 
-![Create ASP.NET Core Web application in Visual Studio](ASP-NET-Core_images/CreateProject.png)
+![Create ASP.NET Core Web application in Visual Studio with File Format DocOI](create-word-in-aspnet-core-web-application_images/file-format-docio-aspnet-core-web-application.png)
```

Comment:
Dont change the previous image folder name. If the Image Folder name has capital letters, you should change to small case, otherwise don't change the previous image folder name.

Check and correct the changes in all places.

Previously I said you to change the image name. Here you used the content in Image Folder. Please ensure properly.

Reply:
Issues resolved